packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -349,7 +353,24 @@ class TextSelectionOverlay {
       OverlayEntry(builder: (BuildContext context) => _buildHandle(context, _TextSelectionHandlePosition.start)),
       OverlayEntry(builder: (BuildContext context) => _buildHandle(context, _TextSelectionHandlePosition.end)),
     ];
-    Overlay.of(context, debugRequiredFor: debugRequiredFor).insertAll(_handles);
+
+    // This method can be call during build phase.
```

Comment:
Yikes. Per me earlier comment: can we not do this?

Reply:
if we make it a widget property, it will require a setstate in the didupdatewidget in textfield, which is also not allowed. or am I missing something?

Reply:
There's no need to call setState() from didUpdateWidget() since it's immediately followed by a build: https://api.flutter.dev/flutter/widgets/State/didUpdateWidget.html